libcontainer/nsenter/nsexec.c

```diff
@@ -543,8 +549,7 @@ void nsexec(void)
 	 */
 	case JUMP_PARENT: {
 			int len;
-			pid_t child;
-			char buf[JSON_MAX];
+			pid_t child, first_child = -1;
```

Comment:
adding `first_child` here is upstream commit e92add2151e50cd66c; I see some parts of that commit in here. I guess it either needs to be brought in (as a separate commit) or not (in this case `first_child` should not appear here).